packages/test/test-utils/src/testFluidObject.ts

```diff
@@ -167,19 +179,26 @@ export class TestFluidObjectFactory implements IFluidDataStoreFactory {
         }
 
         const runtimeClass = mixinRequestHandler(
-            async (request: IRequest) => {
-                const router = await routerP;
-                return router.request(request);
+            async (request: IRequest, rt: FluidDataStoreRuntime) => {
```

Comment:
Nit: `runtime` instead of `rt`, I'd personally prefer `datastoreRuntime`, but I'll leave that choice up to you